src/resources/SearchUsageMetrics/SearchHubs/SearchHubsInterface.ts

```diff
@@ -0,0 +1,50 @@
+export interface RestSearchHub {
+    /**
+     * The name of the search hub
+     */
+    name: string;
+
+    /**
+     * The usage bucket to count queries from this hub into
+     */
+    bucket?: string;
+
+    /**
+     * A description for the search hub
+     */
+    description?: string;
+}
+
+export interface SearchHubModel {
+    /**
+     * The search hub data
+     */
+    hub: RestSearchHub;
+}
+
+export interface SearchHubsList {
+    /**
+     * List of search hub data
+     */
+    hubs: RestSearchHub[];
+}
+
+export interface SearchHubNameParams {
+    /**
+     * The search hub name
+     */
+    hubName: string;
+}
+
+interface SearchHubBucketParams {
+    /**
+     * The search hub bucket
+     */
+    bucket: string;
+}
+
+export type DeleteSearchHubParams = SearchHubNameParams;
```

Comment:
I don't know if that's common practice in the project tbh, but I do not think personally that is a good idea to add type or interface that does nothing.

Reply:
I think you are right, it does not really add anything, I'll remove it.